fix(calendar): leap year time to unix timestamp (#12621)
emqx/emqx: changes/ce/fix-12621.en.md

> Fix rule SQL built-in function `date_to_unix_ts` is incorrect in March and later monthes in leap years.

Reviewer: ```suggestion
Fix incorrect results from rule SQL built-in function `date_to_unix_ts` after March on leap years.
```
